app/grandchallenge/reader_studies/views.py

```diff
@@ -939,7 +985,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ReaderStudyViewSet(ReadOnlyModelViewSet):
+class ReaderStudyViewSet(ModelViewSet):
```

Comment:
Why is this now a ModelViewSet? Doesn't this add create/delete views in the API?